modcc/parser.hpp

```diff
@@ -21,8 +21,9 @@ class Parser : public Lexer {
     expression_ptr parse_integer();
     expression_ptr parse_real();
     expression_ptr parse_call();
-    expression_ptr parse_expression(int prec);
+    expression_ptr parse_expression(int prec, tok t=tok::reserved);
```

Comment:
Defaulting a different stop-token than `tok::eq` means that assignments in subexpressions won't be caught as parse errors; is there a reason why we can't use that default rather than `tok::reserved`?

Reply:
I think using `tok::eq` works too, I will change it.